json/README.md

```diff
@@ -71,7 +71,24 @@ The following example uses **Saxon HE** to convert an OSCAL catalog XML file to
 ```
 java -jar "saxon9he.jar" -xsl:"oscal_catalog_xml-to-json-converter.xsl" -s:"oscal-catalog.xml" -o:"oscal-catalog.json" json-indent=yes
```

Comment:
Perhaps change to `"saxon10he.jar"` or something analogous as Saxon 9.x is now deprecated.

Reply:
@wendellpiez Can you add an issue for upgrading to Saxon 10? Can you reference in that issue that these docs need to be updated?